### ISSUE TITLE
Unique Id checks - id should be in path

### ISSUE THREAD

**Issue by odscjames** [Open-Telecoms-Data/lib-cove-ofds#60]
Currently they return paths of eg `/networks/0/spans/0`

Should they return paths of ef `/networks/0/spans/0/id` ?

**Comment by duncandewhurst** [comment]
Yep, the full path would be more consistent with the results from other checks.
